pyproject.toml

```diff
@@ -42,35 +42,23 @@ classifiers = [
     "Development Status :: 5 - Production/Stable",
     "Environment :: Console",
     "Intended Audience :: Developers",
-    "License :: OSI Approved :: MIT License",
     "Natural Language :: English",
     "Operating System :: OS Independent",
-    "Programming Language :: Python",
```

Comment:
Are these tags not required?

Reply:
poetry attaches relevant Python 3 specific tags automatically. `Programming Language :: Python` is not attached automatically, nor is `Programming Language :: Python :: Implementation :: CPython`. I assume they are not required for anything, but can add them back in, if you like.

Reply:
No, no need, thanks for sharing, I was not aware of this.